resources/helm/v2.2/default/templates/mutatingwebhook.yaml

```diff
@@ -28,7 +28,7 @@ kind: MutatingWebhookConfiguration
 metadata:
   name: istio-revision-tag-default
   labels:
-    maistra-version: "2.2.1"
+    maistra-version: "2.2.2"
```

Comment:
Please run `make gen` and submit these changes in a separate PR, so that they don't pollute the upgrade to 2.3.

Reply:
Good idea, I will do it.